packages/hash/frontend/src/pages/[account-slug]/types/new/entity-type.page.tsx

```diff
@@ -0,0 +1,228 @@
+import { Button, TextField } from "@hashintel/hash-design-system/ui";
+import {
+  Box,
+  Container,
+  formHelperTextClasses,
+  inputLabelClasses,
+  outlinedInputClasses,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { useRouter } from "next/router";
+import { FormEvent, useState } from "react";
+import { useBlockProtocolCreateEntityType } from "../../../../components/hooks/blockProtocolFunctions/ontology/useBlockProtocolCreateEntityType";
+import { useLoggedInUser } from "../../../../components/hooks/useUser";
+import { getPlainLayout, NextPageWithLayout } from "../../../../shared/layout";
+import { TopContextBar } from "../../../shared/top-context-bar";
+import { OntologyChip } from "../entity-type/ontology-chip";
+import { PlaceholderIcon } from "../entity-type/placeholder-icon";
+
+const RequiredText = () => (
+  <Box
+    component="span"
+    color={(theme) => theme.palette.blue[70]}
+    display="inline"
+    fontWeight="bold"
+  >
+    Required
+  </Box>
+);
+
+const HELPER_TEXT_WIDTH = 290;
+
+const Page: NextPageWithLayout = () => {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const router = useRouter();
+  const { user, loading } = useLoggedInUser({
+    onCompleted(data) {
+      if (typeof window !== "undefined") {
+        void router.replace(`/@${data.me.shortname}/types/new/entity-type`);
+      }
+    },
+  });
+
+  if (user && router.query["account-slug"] !== `@${user.shortname}`) {
+    throw new Error("Workspaces not yet supported");
+  }
+
+  const { createEntityType } = useBlockProtocolCreateEntityType(
+    // @todo should use routing URL?
+    user?.accountId ?? ""
+  );
+
+  const [creating, setCreating] = useState(false);
```

Comment:
We also don't need this, since `react-hook-form` provides `formState.isSubmitting`